.charts/overwhelm/values.yaml

```diff
@@ -18,4 +18,6 @@ overwhelm:
       requests:
         cpu: 128m
         memory: 64Mi
+  namespace:
+    create: true
```

Comment:
To be honest, I'm second guessing if we should have that at all, since `helm install --create-namespace ...` already creates the namespace for us...

Reply:
We could take this as a future enhancement OR make the breaking change now. I'm OK with either personally

Reply:
well its less enhancement and more excision ahaha

Reply:
I think we're better off taking care of it now, since otherwise, we'd be adding a parameter that will end up deprecated.

Reply:
To be honest, I'm not 100% sure. We should have a look and see what other big projects are doing with their charts.

Reply:
😁🔪 on it

Reply:
so just remove the whole dang `.charts/overwhelm/templates/overwhelm-system-namespace.yaml`?

Reply:
https://github.com/fluxcd/flux/blob/master/chart/flux/templates/deployment.yaml for instance, it looks like flux just uses the release's namespace.

I think we should take that approach too.

Reply:
Yeah